spark_connect_labextension/handlers/cluster/get_config_bundles.py

```diff
@@ -0,0 +1,36 @@
+"""
+API Route Handler - Get available config bundles
+This file contains the route handler for retrieving config bundles available for use.
+"""
+
+from spark_connect_labextension.handlers.base import SparkConnectAPIHandler
+import tornado
+import json
+
+
+class GetConfigBundlesRouteHandler(SparkConnectAPIHandler):
+    @tornado.web.authenticated
+    def get(self):
+        """
+        GET handler for retrieving Spark config bundles
+
+        :returns: Array of {
+            name: machine-readable config bundle name,
+            displayName: human-readable config bundle name,
+            clusterFilter: Array of strings containing cluster name,
+            options: Array of {
+                name: Spark option name,
+                value: Spark option value
+            }
+        }
+        """
+        bundles = []
+        for bundle_name in self.spark_config_bundles:
+            bundle = self.spark_config_bundles[bundle_name]
+            bundles.append({
```

Comment:
I was thinking it would be nice if all the types in the frontend uses camel case. Some eslint rules enforce this behavior, so I suppose it could be a good practice?

Reply:
@etejedor second opinion